modules/seqexec/web/client/src/main/scala/seqexec/web/client/handlers/WebSocketHandler.scala

```diff
@@ -80,7 +80,7 @@ class WebSocketHandler[M](modelRW: ModelRW[M, WebSocketConnection])
     def onClose(): Unit =
       // Increase the delay to get exponential backoff with a minimum of 200ms and a max of 1m
```

Comment:
The comment still says 200ms.

Reply:
Comments are bad!

Reply:
I think @tpolecat is right. I should remove the comment 😄